packages/drivers/local-driver/src/localDocumentServiceFactory.ts

```diff
@@ -88,8 +88,8 @@ export class LocalDocumentServiceFactory implements IDocumentServiceFactory {
     ): Promise<IDocumentService> {
         ensureFluidResolvedUrl(resolvedUrl);
 
-        const parsedUrl = parse(resolvedUrl.url);
-        const [, tenantId, documentId] = parsedUrl.path ? parsedUrl.path.split("/") : [];
+        const parsedUrl = new URLParse(resolvedUrl.url, {});
```

Comment:
The empty object is intended. From the url-parse docs:

> Note that when url-parse is used in a browser environment, it will default to using the browser's current window location as the base URL when parsing all inputs. To parse an input independently of the browser's current URL (e.g. for functionality parity with the library in a Node environment), pass an empty location object as the second parameter: